src/app/Http/Controllers/Operations/DropzoneOperation.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
Not sure we should have this trait in CRUD... I understand why it's here, and I know we do this already for some stuff... but we're trying to remove all mentions of PRO inside CRUD...

For the new TrashOperation for example, we didn't do this, we asked people to use `Backpack\PRO\Operations\TrashOperation`...

Yeah no, I don't think we should do this. But let's talk about it, all 3 people, in our meeting later today.